documentation/src/pages/index.js

```diff
@@ -128,7 +128,7 @@ function Home() {
 
                      &nbsp;
 
-                     <a href="https://search.maven.org/search?q=kotest">
+                     <a href="https://search.maven.org/search?q=g:io.kotest%20OR%20g:io.kotest.extensions">
```

Comment:
Note: I have chosen a link here which covers the groups `io.kotest` and `io.kotest.extensions`.
However, since the extensions use a different versioning scheme, should this only cover `io.kotest`?